lib/internal/test_runner/utils.js

```diff
@@ -128,10 +128,9 @@ function tryBuiltinReporter(name) {
   return require(builtinPath);
 }
 
-async function getReportersMap(reporters, destinations, rootTest) {
+async function getReportersMap(reporters, destinations) {
   return SafePromiseAllReturnArrayLike(reporters, async (name, i) => {
     const destination = kBuiltinDestinations.get(destinations[i]) ?? createWriteStream(destinations[i]);
-    rootTest.harness.shouldColorizeTestFiles ||= shouldColorize(destination);
```

Comment:
Hey, @MoLow Can you please help me with condition.

In this fix, I have passed testStream (reporter) instead of root to setup(), to avoid any exposure of test objects at userland code. 

Also, where i can add tests for these changes?

Reply:
all tests are passing smoothly!